### DISCUSSION TITLE
WalletConnectModal {"context": "client"} undefined is not a function. React native with expo

### DISCUSSION THREAD

**Discussion by ghost** [WalletConnect/walletconnect-monorepo#2951] [Developer Support]
I'm getting  errors after running the project`
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/115778151/122c7c85-19ee-4b02-af57-7b02b0c5665b)

But after reloading I don't get errors, but getting one when trying to open the walletconnect modal`
{"context": "client"} undefined is not a function.

Any idea why I can get these errors?

Here is my code`

import { View, TouchableOpacity, Text } from 'react-native';
import { SafeAreaView } from 'react-native-safe-area-context';
import { WalletConnectModal, IProviderMetadata, useWalletConnectModal } from '@walletconnect/modal-react-native';
import '@walletconnect/react-native-compat';
import { styles } from './styles';

const App = (): JSX.Element => {

    const { isConnected, provider, open } = useWalletConnectModal();

    const handleButtonPress = async () => {
        if (isConnected) {
            return provider?.disconnect();
        }
        return open();
    };

     const providerMetadata: IProviderMetadata = {
        name: 'test',
        description: 'test',
        url: 'https://walletconnect.com/',
        icons: ['https://avatars.githubusercontent.com/u/37784886'],
        redirect: {
          native: 'test://',
        },
      };
      
       const sessionParams = {
        namespaces: {
          eip155: {
            methods: [
              'eth_sendTransaction',
              'eth_signTransaction',
              'eth_sign',
              'personal_sign',
              'eth_signTypedData',
            ],
            chains: ['eip155:1'],
            events: ['chainChanged', 'accountsChanged'],
            rpcMap: {},
          },
        },
      };
      

    return (
        <SafeAreaView style={styles.container}>
            <View>
                <TouchableOpacity onPress={handleButtonPress}>
                    <Text>Connect</Text>
                </TouchableOpacity>
            </View>
            <WalletConnectModal
                projectId={'my_id'}
                providerMetadata={providerMetadata}
                sessionParams={sessionParams}
            />
        </SafeAreaView>
    );
};

React version 18.2.0
React native version 0.71.8
expo ^48.0.7
@walletconnect/modal-react-native ^1.0.0-rc.3

here is my project [Repo](https://github.com/NarekAv-Solicy/Wallect-connect)

**Comment by ignaciosantise** [comment]
Discussion moved here: https://github.com/WalletConnect/modal-react-native/issues/27
